CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1442,23 +1549,22 @@ do -- Search stuff
 	end
 
 	local function detectGamepad(input, processed)
-		if input.UserInputType == Enum.UserInputType.Gamepad1 then
+		if input.UserInputType == Enum.UserInputType.Gamepad1 and not UserInputService.VREnabled then
 			searchFrame.Visible = false
 		else
 			searchFrame.Visible = true
 		end
 	end
-	local uis = game:GetService("UserInputService")
-	uis.InputBegan:connect(detectGamepad)
-	uis.InputChanged:connect(detectGamepad)
+	UserInputService.InputBegan:connect(detectGamepad)
```

Comment:
I will update that; I am guessing that when this was written we didn't have this API.